### DISCUSSION TITLE
New keyboard shortcut Ctrl + Shift + S for "Save As..."?

### DISCUSSION THREAD

**Discussion by abrahamwolk** [ControlSystemStudio/phoebus#2734] [Ideas]
Ctrl + S is currently a keyboard shortcut for "Save". Would it make sense to also add the keyboard shortcut Ctrl + Shift + S for "Save As..."? It could be added to the Display Builder and to the Data Browser.

It is a standard shortcut in many applications.

**Comment by kasemir** [comment]
Sure, just note that Mac OS uses "Command S" for save, and "Command Shift S" for save-as.

So would need to change the exact key combination similar to https://github.com/ControlSystemStudio/phoebus/blob/7e26b3c9cfa5a3ed8a1dc3c655dcd8a03fa8aeac/core/ui/src/main/java/org/phoebus/ui/application/PhoebusApplication.java#L226-L228

**Comment by kasemir** [comment]
I'm actually not sure where the "Ctrl S" is currently mapped. It might not be in the data browser, display builder, scan editor, ... but at a higher level in core-ui, allowing to "save" any `DockItemWithInput` that `isDirty` or `isSaveAsSupported`

**Comment by abrahamwolk** [comment]
Thank you for the information! I will go ahead and implement the Ctrl/Command + Shift + S functionality then.
